jerry-core/ecma/operations/ecma-regexp-object.h

```diff
@@ -30,6 +30,7 @@
 
 /**
  * RegExp flags
+ * Should be kept in sync with jerry_regexp_flags_t in jerryscript-core.h
```

Comment:
```suggestion
 *
 * Note:
 *      This enum has to be kept in sync with jerry_regexp_flags_t.
```
Better use existing style, e.g., as at `jerry_debugger_output_subtype_t`. (Empty line + `Note:` + indentation.)